hnix.cabal

```diff
@@ -401,8 +402,9 @@ library
     , gitrev >= 1.1.0 && < 1.4
     , hashable >= 1.2.5 && < 1.4
     , hashing >= 0.1.0 && < 0.2
-    , hnix-store-core >= 0.1.0 && < 0.3
-    , http-client >= 0.5.14 && < 0.6 || >= 0.6.4 && < 0.8
+    , hnix-store-core >= 0.3.0 && < 0.4
```

Comment:
@layus if you wouldn't mind updating these bounds for the latest release, perhaps @expipiplus1 would be happy to merge this?